pkgs/tools/misc/bat-extras/default.nix

```diff
@@ -112,7 +112,7 @@ let
       dontBuild = true; # we've already built
 
       doCheck = true;
-      nativeCheckInputs = [ bash fish zsh ] ++ (lib.optionals stdenv.isDarwin [ getconf ]);
+      nativeCheckInputs = [ bat bash fish zsh ] ++ (lib.optionals stdenv.isDarwin [ getconf ]);
```

Comment:
When I was testing the new config in my local environment:

```
nix-build -A bat-extras ~/dev/me/github.com/nixos/nixpkgs
```

I get the `bat was not found on $PATH` error in the tests in L118. Adding `bat` here resolves the issue.